db/data/20231109143922_set_up_bangladesh_december_experiment.rb

```diff
@@ -0,0 +1,101 @@
+# frozen_string_literal: true
+
+class SetUpBangladeshDecemberExperiment < ActiveRecord::Migration[6.1]
+  START_TIME = DateTime.parse("1 Dec 2023").beginning_of_day
+  END_TIME = DateTime.parse("31 Dec 2023").end_of_day
+  CURRENT_PATIENTS_EXPERIMENT = "Current Patient December 2023"
+  STALE_PATIENTS_EXPERIMENT = "Stale Patient December 2023"
+  PATIENTS_PER_DAY = 5000
+  facility_slugs = {
+    basrishal_facilities: ["uhc-agailjhara", "uhc-babuganj-facility", "uhc-bakerganj", "uhc-hijla", "uhc-mehendiganj", "uhc-muladi", "uhc-wazirpur"],
+    jhalokathi_facilities: ["uhc-kathalia-86845c14-b23d-4578-820a-63eb9628422a", "uhc-nalchity-97cb9397-8ca7-4edb-81ae-2ac8fb1ea93a", "uhc-rajapur-40d0e15a-37c4-4833-8f55-446bb210c457"],
+    dhaka_facilities: ["thc-tejgaon-sadar", "uhc-dhamrai", "uhc-dohar", "uhc-nawabganj"],
+    shariatpur_facilities: ["uhc-damuddya", "uhc-goshairhat"],
+    munshiganj_facilities: ["uhc-sreenagar", "uhc-tongibari", "uhc-sirajdikhan"],
+    chattogram_facilities: ["uhc-hathazari", "uhc-fatikchori", "uhc-anwara", "uhc-satkania", "uhc-lohagara", "uhc-patiya", "uhc-rangunia", "uhc-sitakunda", "uhc-chandanaish", "uhc-raozan", "uhc-banshkhali", "uhc-boalkhali", "uhc-karnaphuli", "uhc-mirsharai", "uhc-sandwip"],
+    feni_facilities: ["uhc-fulgazi", "uhc-parsuram", "uhc-daganbhuiya", "uhc-sonagazi", "uhc-chhagalniya"],
+    netrokona_facilities: ["uhc-atpara", "uhc-barhatta", "uhc-kalmakanda", "uhc-kendua", "uhc-khaliajuri", "uhc-madan", "uhc-mohanganj", "uhc-purbadhala", "uhc-durgapur-dbd6efea-8950-410e-abc3-ed9a329fb9d3"],
+    sirajganj_facilities: ["uhc-belkuchi", "uhc-chowhali", "uhc-kamarkhanda", "uhc-kazipur", "uhc-shahzadpur", "uhc-tarash", "uhc-ullapara"],
+    pabna_facilities: ["uhc-bera", "uhc-bhangura", "uhc-faridpur", "uhc-iswardi", "uhc-santhia", "uhc-chatmohar", "uhc-sujanagar-4b973cb1-d584-4100-91ca-778f218b502e", "atgharia-upazila-health-complex"],
+    jamalpur_facilities: ["uhc-islampur", "uhc-melandah", "uhc-sarishabari", "uhc-dewanganj", "uhc-bakshiganj", "uhc-madarganj"],
+    sherpur_facilities: ["uhc-jhenaigati", "uhc-nalitabari", "uhc-nakhla", "uhc-sribordi"]
+  }.values.reduce(:+)
```

Comment:
This is good way of representing data in the expected format, and having readability. Better than using inline comments. Good one!